extensions/amp-experiment/0.1/amp-experiment.js

```diff
@@ -28,8 +28,8 @@ const ATTR_PREFIX = 'amp-x-';
 export class AmpExperiment extends AMP.BaseElement {
 
   /** @override */
-  isLayoutSupported(unusedLayout) {
-    return true;
+  isLayoutSupported(layout) {
+    return layout == Layout.NODISPLAY || layout == Layout.CONTAINER;
```

Comment:
could you explain a little bit why is this change needed?
This was to mimic `amp-analytics`'s behavior.

If we want to make sure it's never displayed, we can probably explicitly set the element to hidden in `buildCallback`.